ssu.1

```diff
@@ -40,6 +40,10 @@ run interactive shell
 preserve environment
 .It Fl e Ar file
 edit file
+.It Fl n
+treat
+.Em user
```

Comment:
If you wanted to be really consistent and specific about semantic markup, use `.Ar user`

Reply:
Done, thanks!